src/website/src/pages/ec/components/list/docs/usage.mdx

```diff
@@ -0,0 +1,146 @@
+---
+title: Usage
+order: 1
+---
+
+import { Paragraph } from '@ecl/website-components';
+
+<Paragraph size="lead">
+  Lists group together related pieces of information so they are clearly
+  associated with each other and easy to read. Lists are good from a structural
+  point of view as they help create a well-structured and more accessible
+  content. Semantically correct lists help visitors read your web site.
+</Paragraph>
+
+## Anatomy
+
+Lists or "typography lists" are optimized for reading comprehension.
```

Comment:
```suggestion
Lists or "typography lists" are optimized for reading comprehension.
```
Lists are used to achieve easy scalability.

Reply:
Bonjour, it is too slow to edit it line by line. When a long sentence is broken down to multiple lines...may I use the content but to edit it in a different page?